source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -28,5 +28,76 @@ more naturally readable.
 Operations
 ----------
 
+Each of the following sections provides information and examples for
+aggregation expression operations available in the {+driver-short+}.
+This guide sorts these operations into categories based on general
+purpose and functionality.
+
+.. important::
+
+   The Query API expression that the driver generates from each
+   aggregation expression operation might not be
+   identical to the Query API expression provided in each example.
+   However, they are functionally equivalent.
+
+Boolean Expressions
+~~~~~~~~~~~~~~~~~~~
+
+You can invoke boolean operations on boolean values of type ``MqlBoolean``.
+
+The following table describes the boolean aggregation operation methods
+available in the {+driver-short+} and the equivalent expressions in the
+Query API:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Java Aggregation Expression
```

Comment:
changed to just Aggregation Method

Reply:
The current naming (in the Manual) is confusing to me, so not sure I can point you to the right option.

However, I wonder if Aggregation Method might generate confusion since that term is used to categorize "aggregate()" and "mapReduce()" in the Manual here: https://www.mongodb.com/docs/v6.0/reference/operator/aggregation/interface/#aggregation-methods

On the other hand, those are also called Aggregation Commands here:
https://www.mongodb.com/docs/v6.0/reference/aggregation-commands-comparison/#aggregation-commands-comparison-table

Since this page refers to them as "Aggregation Expression Operators", maybe that would be the best choice? Or qualify them with "Java", e.g. "Java Aggregation Expression Operators".

Reply:
chose "Java Aggregation Expression Operator"